### PR TITLE
sql checks

### PR DESCRIPTION
**Pull request by usfalami**

### CODE REVIEW

**Comment by Silicoman** [comment]
This change can also be applied easly inside php and python plugin.

**Comment by Silicoman** [comment]
This change can also be applied easly inside php and python plugin.

File: src/java-plugin/src/test/files/AvoidFullSQLRequestCheck.java [hunk 1]
```diff
@@ -1,12 +1,17 @@
 package fr.cnumr.java.checks;
 
+import java.util.regex.Pattern;
```
Comment (MP-Aubay) [inline src/java-plugin/src/test/files/AvoidFullSQLRequestCheck.java:3]: Seems not used

File: src/java-plugin/src/test/files/AvoidFullSQLRequestCheck.java [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
+
 class AvoidFullSQLRequestCheck {
```
Comment (MP-Aubay) [inline src/java-plugin/src/test/files/AvoidFullSQLRequestCheck.java:5]: Maybe adding some test like that would be wised : 
`dummyCall("SELECT user FROM (SELECT * FROM table)");`

**Review by MP-Aubay — COMMENTED** [review]
Adding some review, hope it can help

**Pull request merged**